### PR TITLE
You have to open mech panel first to attach the mecha tracker.

### PR DESCRIPTION
**Pull request by paganiy**
## About The Pull Request
As stated in the title, now in order to attach the tracker to the mech, you must first open its panel with a screwdriver.
## Why It's Good For The Game
Is it a good thing that a mecha can be destroyed with an tiny item that attaches in 1 click no matter what?
## Changelog
:cl:
balance: You have to open mech panel first to attach the mecha tracker.
/:cl:

### CODE REVIEW

**Comment by GPeckman** [comment]
I opened a PR that did the same thing a while ago, #79041. But apparently its intended for the tracking beacon to be installed easily.

**Review by ZephyrTFA — CHANGES_REQUESTED** [review]
I don't like this.
This is soul.
Many before you have tried to do this.

**Comment by paganiy** [comment]
> I don't like this. This is soul. Many before you have tried to do this.

sir, this is "power-creep"

**Comment by SmArtKar** [comment]
> > I don't like this. This is soul. Many before you have tried to do this.
> 
> sir, this is "power-creep"

thats.... not the definition of power creep, at all.

**Comment by ZephyrTFA** [comment]
> > I don't like this. This is soul. Many before you have tried to do this.
> 
> sir, this is "power-creep"

It has been this way since inception. Do you not know what power creep even means

**Comment by paganiy** [comment]
what is the soul in the item that allows you to kill in melee in 1 click mech that ate half of the station's resources + 9999 RND research points, i don't understand it

**Comment by ZephyrTFA** [comment]
This is only a soft block, any maintainer can still merge this if they want it.

Sovl

**Comment by githubuser4141** [comment]
> The fact that you can attach these so easily is why I was convinced to not make combat mechs auto-start with one.
> 
> I have regretted that ever since.
> 
> Anyway if you're doing a mech rampage just destroy the console first. 99% chance no one ever builds another.

Tbh, haven't mechs been nerfed pretty heavily with the new RND changes + not being able to punch items on the floor, that they can probably be buffed by removing BS counters like 'stick EMP beacon on when it's moving'? Not only the ability to stick one on, you can **Stack** mech beacons and mech consoles can be printed anywhere.

**Comment by zxaber** [comment]
Stacking them is either a bug or an unintended oversight and should be removed anyway.

There's a lot of hyperbole here. The beacons are not a one-click win, not on their own. You require someone at a console, who presses the button constantly in-between pulse cooldowns. The cooldowns for EMP pulses are intentionally longer than the mech weapon disable timer, so it doesn't fully stop you from pounding their face in (even if it does make it more difficult). EMP damage is not devastating for mechs because the main point is interrupting weapons fire. Though it could be tuned down a tad more if part of an overall balance package.

The biggest potential problem is maint consoles, which tend to be an issue (or at least a concern) across a number of similar features. Best bet here would probably be a message to the pilot(s) about why the mech was EMPd, and *where* the console that did it is. Outside of that, if you're being attacked by two people coordinating together, and you can't stop the console user because his buddy is forcing a fight in the halls, it's just standard counterplay.

My take on proper balance would be making combat mechs (sans Paddy and Syndicate) force a beacon automatically into the mech upon completion, cutting down the EMP damage slightly to allow not-busy mechs time to hunt down the console, giving an overlay animation to mechs hit by any EMP as a visual "free to attack" indicator for opponents, and giving mech pilots hit by a console EMP the room name of the console as stated above. The overall idea here is to make more standard the idea of the exosuit console being a proper tool against rogue mechs, while not making it a straight win button alone. For the traitor robos, a hidden weapons bay on a ripley remains a valid option to get around the beacon weakness. Paddy also remains as a (permanently not space worthy) option.

By the by, the R&D changes haven't really been anti-mech at all. Worst thing really is that basic ballistics (Scattershot and AC 2) are behind toxins research, which I don't really agree with but it's not like the end of the world.

**Comment by githubuser4141** [comment]
> By the by, the R&D changes haven't really been anti-mech at all. Worst thing really is that basic ballistics (Scattershot and AC 2) are behind toxins research, which I don't really agree with but it's not like the end of the world.

LMG and scattershot are the most valuable mech guns in the game, locking them behind toxins research (which few people do) is pretty significant, it means people can't just grab it after the mech experiments as they used to be able to do.

It's not the only R&D change that affects mechs, mech plasma cutters which are used to speedrun mining by roboticists are also locked behind plasma shells, meaning for the average user - you are forced to rely on a proper research/logistics network to actually get a combat mech. You can't just one-man mining with a Ripley, come back and scan your own Ripley and presto you have a Durand anymore.

Anyway, at the moment beacons do about 20 damage to mechs which is enough to kill them in 20 or less hits, rolls for a malfunction (malfunction damage minimum is 15) and drains 10-15% power. If stacking is fixed and mechs are told where the EMP'er is located it's still a pretty huge anti-mech play, especially if mech melee is chanaged to require a melee tool (hammer, otherwise it can't break walls) to melee stuff. In the sense that a gun mech literally wouldn't be able to reach a EMP console behind a wall at all.

**Comment by JohnFulpWillard** [comment]
I do not agree with this PR either @ZephyrTFA though I also always wanted mechs in general to be moved to Lavaland stuff

**Pull request closed without merging**
